src/seedu/addressbook/AddressBook.java

```diff
@@ -348,7 +348,7 @@ private static void loadDataFromStorage() {
     public static String executeCommand(String userInputString) {
         final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
         final String commandType = commandTypeAndParams[0];
-        final String commandArgs = commandTypeAndParams[1];
+        final String commandArgs = commandTypeAndParams[1].toLowerCase(); // to make the find function case insensitive
```

Comment:
that's all it takes: turning the input keyword to lowercase as long as you do the same normalisation when retrieving the data from the addressbook for comparison. But is this a good place to make this change? is it obvious to someone looking at your entire codebase the first time?